pkg/apis/serving/v1beta1/inference_service.go

```diff
@@ -66,6 +67,54 @@ type LoggerSpec struct {
 	MetadataHeaders []string `json:"metadataHeaders,omitempty"`
 }
 
+type ScalerSpec struct {
+	// Minimum number of replicas, defaults to 1 but can be set to 0 to enable scale-to-zero.
+	// +optional
+	MinReplicas *int `json:"minReplicas,omitempty"`
+	// Maximum number of replicas for autoscaling.
+	// +optional
+	MaxReplicas int `json:"maxReplicas,omitempty"`
```

Comment:
Maybe we can still leave these two fields out as users are already getting used to the existing locations, so we do not have to duplicate them.